.github/workflows/core-cicd-tests.yml

```diff
@@ -11,15 +11,12 @@ jobs:
     name: Unit Tests
     runs-on: ubuntu-18.04
     env:
-      DOT_CICD_BRANCH: master
```

Comment:
Let's leave this variable since we need to modify it in the create-release-branch script